onnxruntime/test/shared_lib/test_inference.cc

```diff
@@ -439,6 +441,40 @@ TEST(CApiTest, create_session_without_session_option) {
 }
 #endif
 
+#ifdef REDUCED_OPS_BUILD
+TEST(ReducedOpsBuildTest, test_model_with_reduced_ops_build) {
+  constexpr PATH_TYPE model_uri = TSTR("testdata/test_reduced_ops_build_model");
+  std::vector<Input> inputs(1);
```

Comment:
Would be helpful for the name to say 'what' is being tested. Suggest renaming the model and test so that's clearer.

Reply:
Renamed tests like:

- test_included_ops
- test_excluded_ops

Reply:
Comments added for better understanding.